src/components/calcite-input/calcite-input.e2e.ts

```diff
@@ -862,5 +862,111 @@ describe("calcite-input", () => {
       expect(await calciteInput.getProperty("value")).toBe(initialValue);
       expect(await input.getProperty("value")).toBe(initialValue);
     });
+
+    it(`disallows pasting just text characters with no initial value`, async () => {
+      const page = await newE2EPage({
```

Comment:
When pasting, if internally the component is the same when it starts with an initial value in the HTML or not, I think you can merge the with-initial/without-initial set of tests.